cmd/shortener/main_test.go

```diff
@@ -111,14 +115,80 @@ func Test_shortURL(t *testing.T) {
 			gin.SetMode(gin.TestMode)
 			w := httptest.NewRecorder()
 
-			storage := store.NewStorage()
+			storage, err := store.NewStorage("./test.json")
+			if err != nil {
+				panic(err)
```

Comment:
не надо паниковать даже в тестах: лучше:

```suggestion
				t.Errorf("failed to initialize a new storage: %v", err)
				return
```